test/jdk/java/awt/print/bug8023392.java

```diff
@@ -84,15 +113,15 @@ public SimplePrint2() {
                 String s = "3a) a b c d e                                     ";
                 @Override
                 protected void paintComponent(Graphics g) {
-                    sun.swing.SwingUtilities2.drawChars(this, g, s.toCharArray(),
+                    SwingUtilities2.drawChars(this, g, s.toCharArray(),
```

Comment:
cant we use `Graphics.drawChars` instead in all places `SwingUtilities2.drawChars` is used?

Reply:
`Graphics.drawChars` isn't a static method, which will require additional changes to this test. Is it preferred to use `Graphics` over the `SwingUtilities2` version?

Reply:
This test is specifically about testing the spacing of text in Swing printing.
I think this SwingUtilities call isn't just "the way he happens to be printing text",
This call is the very thing under test ! And note that it takes a Component parameter
which affects how that call renders the text. It is used to figure out the FRC to use.
So changing this to use Graphics would be a problem.